.github/workflows/e2e.yaml

```diff
@@ -93,10 +93,10 @@ jobs:
       - name: Load provider image
         run: sudo docker load -i provider-images.tar
       - name: Load k8s-snap old image
-        if: matrix.ginkgo_focus == 'Workload cluster upgrade'
         run: |
           sudo docker load -i k8s-snap-image-old.tar
       - name: Load k8s-snap new image
+        if: matrix.ginkgo_focus == 'Workload cluster upgrade'
```

Comment:
all non-upgrade images use the `k8s-snap:dev-old` image. I introduced this bug in the last PR